src/module/data/item/ability.mjs

```diff
@@ -165,6 +165,8 @@ export default class AbilityModel extends BaseItemModel {
     context.keywords = Object.entries(config.keywords).map(([value, {label}]) => ({value, label}));
     context.actionTypes = Object.entries(config.types).map(([value, {label}]) => ({value, label}));
 
+    context.triggeredAction = ["triggered", "freeTriggered"].includes(this.type);
```

Comment:
I would rather have this as an optional Boolean on the config object, same level as the label, something like `triggered: true`.

Reply:
I just made the changes. I think this is what you were asking. I assumed the issue is the hard coding of what "triggered" is being in the context instead of the config.

Reply:
Also, I think I did the typing right? Not terribly familiar with that.